content/guides/core-concepts/cypress-app.md

```diff
@@ -6,29 +6,109 @@ title: Cypress App
 
 ## <Icon name="graduation-cap"></Icon> What you'll learn
 
-- The names and purposes of the visual parts of the Cypress App
+- The names and purposes of the visual parts of the Cypress app
 - How to use the Selector Playground for targeting your page elements
-- How to debug tests using the built-in features of the Cypress App
+- How to debug tests using the built-in features of the Cypress app
 
 </Alert>
 
 ## Overview
 
-Cypress runs tests in a unique interactive application that allows you to see
-commands as they execute while also viewing the Application or Component Under
-Test.
+The Cypress desktop app has powerful features for creating, configuring,
+browsing, and most importantly running your tests. We'll touch on a few of the
+different functions before moving on to the Test Runner, the heart of the
+Cypress testing experience.
 
-<DocsImage src="/img/guides/cypress-app/gui-diagram.png" alt="Cypress App"></DocsImage>
+<Alert type="info">
+
+<strong class="alert-header">Dashboard Integration</strong>
+
+When you configure Cypress to record tests to the
+[Dashboard](/guides/dashboard/introduction) you'll see data from your latest
+recorded runs in the Cypress app, increasing visibility into your test history.
+If you're not sure whether you're logged in, you can always check using the
+profile control in the top right of the window.
+
+The data used by these features is pulled from the Dashboard, so they will only
+reflect test results that were
+[recorded](/guides/dashboard/projects#Set-up-a-project-to-record).
+
+</Alert>
+
+### The Launchpad
+
+<DocsImage src="/img/guides/core-concepts/cypress-app/the-launchpad.png" alt="The Launchpad"></DocsImage>
+
+The Launchpad is your portal to Cypress, helping with onboarding, choosing a
+testing type and launching a browser.
+
+Once you get comfortable with Cypress you might find you don't need the
+Launchpad any more. In this case
+[you can run `cypress open` with the `--browser` and `--component` or `--e2e` options](/guides/guides/command-line#cypress-open)
+to go straight to the Spec Explorer.
+
+### The Spec Explorer
+
+TODO Image with rest of columns
+<DocsImage src="/img/guides/core-concepts/cypress-app/spec-explorer.png" alt="The Spec Explorer"></DocsImage>
+
+On choosing your browser in the Launchpad, you'll be presented with a list of
+your specs, their names, locations, and data from your latest recorded runs.
+Here you can launch specs by clicking them, create new blank or example specs,
+or search specs by name (handy for large test suites).
+
+We've made the following spec metadata available directly in the Spec Explorer:
+
+#### Last updated
+
+This column shows when the code of this spec was last updated, using your local
+system's own metadata. This metadata will vary from machine to machine, as it
+reflects the state of the code as stored on that computer.
```

Comment:
This should definitely call out that 'Last updated' uses git if available in your project, and falls back to system data otherwise.